.pre-commit-config.yaml

```
@@ -75,6 +75,41 @@ repos:
   hooks:
   - id: forbid-html-img-without-alt-text
 
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: 'v0.910-1'
+  hooks:
+  - id: mypy
+    additional_dependencies:
+    - types-six
+    - types-backports
+    - types-pyOpenSSL
+    - types-requests
+    args:
+    - --install-types
+    - --non-interactive
+    - --python-version
+    - '3.5'
```

I wonder if this would work:
```suggestion
    - --python-version=3.5
```